internal/core/legacy/services/filesvc/download.go

```diff
@@ -37,6 +38,19 @@ const (
 	headerContentType        = "Content-Type"
 	headerContentDisposition = "Content-Disposition"
 	HeaderContentLength      = "Content-Length" // The Content-Length entity header indicates the size of the entity-body, in bytes, sent to the recipient.
+
+	headerContentTypePng = "image/png"
+)
+
+var (
+	// allowedContentTypes is a map of allowed file code(key) and content type(value).
+	allowedContentTypes = map[string]string{
```

Comment:
any doc link for these file header code? please link it.

Reply:
added